app.rb

```diff
@@ -1,7 +1,25 @@
 require 'sinatra/base'
+require './lib/game_choice.rb'
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  enable :sessions
+
+  get '/' do
+    erb :index
+  end
+
+  post '/names' do
+    @player_name = params[:name]
+    erb :options
+  end
+
+  get '/names' do
```

Comment:
Nice. It's common practise for endpoints receiving data from a form to be a POST request which redirects to a GET request.